docs/provider/vagrant.rst

```diff
@@ -26,7 +26,7 @@ example the following is a valid configuration
    :linenos:
 
 The list of the sizes may be found :enos_src:`here
-<enos/provider/enos_vagrant.py#L12>`.
+<https://gitlab.inria.fr/discovery/enoslib/blob/master/enoslib/infra/enos_vagrant/constants.py#L23-48>`.
```

Comment:
I haven't take a look at the output, but this seems wrong to me.

`enos_src` is a markup  as defined in the conf.py 

https://github.com/BeyondTheClouds/enos/blob/82d0615b4ddffdc81d394d4a895bef1add705473/docs/conf.py#L85-L96